config/global-constants/src/lib.rs

```diff
@@ -9,7 +9,7 @@
 #![forbid(unsafe_code)]
 
 /// Definitions of global cryptographic keys (e.g., as held in secure storage)
-pub const CONSENSUS_KEY: &str = "consensus";
+pub const CONSENSUS_KEY: &str = "consensus_v2";
```

Comment:
why are we introducing a new key?

Reply:
Let's handle this as a separate migration.

Reply:
> why are we introducing a new key?

@davidiw, as discussed offline, this was because vault doesn't allow a key to be non-exportable after previously being exportable (see the PR description).

> Let's handle this as a separate migration.

Okay, thanks @mgorven, sounds good, I'll update the PR.